app/adapters/contributor.ts

```diff
@@ -1,20 +1,21 @@
+import DS from 'ember-data'; // eslint-disable-line no-unused-vars
```

Comment:
This should be taken care of in eslint with https://github.com/nzakas/eslint-plugin-typescript/blob/master/docs/rules/no-unused-vars.md

Reply:
Please `yarn add --dev @types/ember-data` and you won't need this.